q2_cutadapt/_examples.py

```diff
@@ -0,0 +1,30 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2022, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+def cutadapt_demux_single(use):
+    sequence_url = ('https://qiime2-data.s3.us-west-2.amazonaws.com/'
```

Comment:
```suggestion
    sequence_url = ('https://data.qiime2.org/'
```
nice little URL shortcut for this 🙂